### ISSUE TITLE
Single file attachment fails A11y test when the label differs from the button text

### ISSUE THREAD

**Issue by Blackbaud-AlexKingman** [blackbaud/skyux-forms#222]
Create a single file attachment component, include a `sky-file-attachment-label` that contains anything other than `Choose file` and your A11y tests will fail:

https://dequeuniversity.com/rules/axe/3.5/label-content-name-mismatch?application=axeAPI

We don't necessarily need the `labeled-by` attribute because the button will always have text.

[AB#1614678](https://blackbaud.visualstudio.com/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/1614678)
